pkg/workflow/tasks/discover.go

```diff
@@ -68,10 +70,20 @@ func (td *taskDiscover) GetTaskGenerator(ctx context.Context, name string) (type
 }
 
 func suspend(step v1beta1.WorkflowStep, opt *types.GeneratorOptions) (types.TaskRunner, error) {
-	return &suspendTaskRunner{
-		id:   opt.ID,
-		name: step.Name,
-	}, nil
+	tr := &suspendTaskRunner{
+		id:    opt.ID,
+		name:  step.Name,
+		delay: false,
```

Comment:
I think it's quite confusing we use `delay` here, since what we do is auto continue the workflow after some time. We can change the `delay` to `wait` or `autoContinue` I guess?

Reply:
ok, it's good. I'll change it.